features/shopcarts.feature

```diff
@@ -22,12 +22,19 @@ Scenario: List all shopcarts
 Scenario: Create a Shopcart
     When I visit the "Home Page"
     And I set the Shopcart "user_id" to "1"
-    And I click the "Add Products" button
-    And I add "1" "Apple" to the cart
+    And I click the "Add-Products" button
+    And I add "2" of Product "3" to the cart
     And I click the "Create" button
     Then I should see Shopcart "1" in the results
     And I should not see "Status Code: 409. Shopcart for user 1 already exits" in the form
 
+Scenario: Visit an existing Shopcart
+    When I visit the "Home Page"
+    And I visit Shopcart "1"
+    Then I should see "1" in the header
+    And I should not see "404 Not Found"
+
+@slow
```

Comment:
We don't need the `slow` decorator now do we? Same on line 47.

Reply:
This helps when you run tests you can filter the ones with a slow tag so it was useful.